src/libraries/Microsoft.PowerFx.Json/Public/Types/UserProvidedTypeDefinitions/FormulaTypeToSchemaHelper.cs

```diff
@@ -51,12 +51,12 @@ private static FormulaTypeSchema ToSchema(FormulaType type, DefinedTypeSymbolTab
 
             if (type._type.AggregateHasExpandedType())
```

Comment:
> (type._type.AggregateHasExpandedType()) [](http://example.com/codeflow?start=14&length=40)

nit: 
is (type is AggreagtedType aggType) {
  var logicalName = aggType.TableSymbolName
}